### PR TITLE
doc: Trusted Storage maturity

### PR DESCRIPTION
**Pull request by SebastianBoe**
Trusted Storage maturity documentation.

### CODE REVIEW

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 1]
```diff
@@ -255,6 +255,7 @@ The following sections contain the tables indicating the software maturity level
 * PSA Crypto
 * |NSIB|
 * Hardware Unique Key
+* Trusted Storage
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/software_maturity.rst:258]:
```suggestion
* Trusted storage
```

File: doc/_scripts/software_maturity/software_maturity_features.yaml [hunk 2]
```diff
@@ -134,6 +134,8 @@ features:
     Immutable Bootloader as part of build: SECURE_BOOT
   hw_unique_key:
     Key Derivation from Hardware Unique Key: HW_UNIQUE_KEY
+  trusted_storage:
+    Trusted Storage implements Secure Storage without using TF-M: TRUSTED_STORAGE
```
Comment (mia-ko) [inline doc/_scripts/software_maturity/software_maturity_features.yaml:138]:
```suggestion
    Trusted storage implements the PSA Certified Secure Storage APIs without using TF-M: TRUSTED_STORAGE
```

File: doc/nrf/releases_and_maturity/software_maturity.rst [hunk 3]
```diff
@@ -284,6 +285,13 @@ Hardware Unique Key
 
   .. sml-table:: hw_unique_key
 
+Trusted Storage
```
Comment (mia-ko) [inline doc/nrf/releases_and_maturity/software_maturity.rst:288]:
```suggestion
Trusted storage
```

**Review by mia-ko — CHANGES_REQUESTED** [review]

File: doc/_scripts/software_maturity/software_maturity_features.yaml (hunk 2 above)
Comment (mia-ko) [reply doc/_scripts/software_maturity/software_maturity_features.yaml:138]: I don't see the change reflected in the file.

**Review by mia-ko — APPROVED** [review]

File: doc/_scripts/software_maturity/software_maturity_features.yaml [hunk 4]
```diff
@@ -134,6 +134,9 @@ features:
     Immutable Bootloader as part of build: SECURE_BOOT
   hw_unique_key:
     Key Derivation from Hardware Unique Key: HW_UNIQUE_KEY
+  trusted_storage:
+    Trusted storage implements the PSA Certified Secure Storage APIs without using TF-M:
+    TRUSTED_STORAGE
```
Comment (mia-ko) [inline doc/_scripts/software_maturity/software_maturity_features.yaml:139]: Btw, shouldn't this be in line 138?

**Review by SeppoTakalo — APPROVED** [review]

**Pull request merged**